README.md

```diff
@@ -136,6 +136,20 @@ First - ensure that the following environment variables are set to the appropria
 
 NOTE: These values (and keys) need to be shared/common across all microservices that use each common authenticator host. If any of the environment *keys* for each of these attributes needs to be modified these can be reconfigured in fsd_utils/authentication/config.py.
 
+To generate new keys, you use the following commands:
+(Note: be aware of where your keys get created, when setting these keys as envvars in cf you'll need to be in the directory where you created them).
+```bash
+openssl genrsa -out private_key.pem 2048
+openssl rsa -pubout -in private_key.pem -out public_key.pem
+```
+
+For dev/test environments, these have been set manually with the following:
+```bash
+cf t -s sandbox
+cf set-env {app} RSA256_PRIVATE_KEY_BASE64 $(cat private_key.pem | base64)
+cf set-env {app} RSA256_PUBLIC_KEY_BASE64 $(cat public_key.pem | base64)
+```
+
```

Comment:
This isn't accurate.  We use github actions and secrets to deploy/store these keys.  Update this tomorrow morning.

Reply:
Did this get updated?

Reply:
Nope will do in a bit!